### DISCUSSION TITLE
wdyk rcbo

### DISCUSSION THREAD

**Discussion by adrieSuryaK** [jasonacox/tinytuya#401] [Q&A]
excuse me sir, I got a bit confusing mapping in wdyk rcbo...
Does anyone know where the current, voltage and load data is?

{
  "result": true,
  "status": {
    "power": true,
    "temp": 41,
  },
  "dps": {
    "1": true,
    "9": 0,
    "26": 0,
    "38": "2",
    "40": false,
    "104": 0,
    "105": 41,
    "107": 0,
    "114": 6969,
    "115": "400100000008        ",
    "117": false,
    "118": 0
  }
}

thx

**Comment by uzlonewolf** [comment]
I'm not seeing anything which looks like a voltage.  It may need an "update DPs" command to get them, or it may just broadcast them asynchronously.  Are you using a recent version of TinyTuya?  It should ask you if you want to download the DP mapping, after which it'll be included in devices.json.
